spec/e2e/end_to_end.rb

```diff
@@ -109,9 +109,9 @@ def transfer_test(imported_address, new_address)
   puts "Transferred 1 Gwei from #{imported_address} to #{new_address}"
 
   # Transfer some eth for gas fee to new address.
-  t2 = imported_address.transfer(0.00000003, :eth, new_address).wait!
+  t2 = imported_address.transfer(0.00001, :eth, new_address).wait!
```

Comment:
We should add a monitor on the balance for this address just so we can be forewarned (or maybe add a faucet call in the e2e test)